scripts/linters/html_linter.py

```
@@ -305,10 +305,9 @@ def handle_data(self, data: str) -> None:
             data: str. Contents of HTML file to be parsed.
         """
         data_lines = data.split('\n')
-        opening_block = tuple(
-            ['{% block', '{% macro', '{% if', '% for', '% if']
-        )
-        ending_block = tuple(['{% end', '{%- end', '% } %>'])
+        opening_block = tuple(['{% block', '{% macro', '{% if'])
```

Ruff (0.15.15)

[warning] Unnecessary list literal passed to `tuple()` (rewrite as a tuple literal)

Rewrite as a tuple literal

(C409)